lib/modules/job/providers/jobs_dashboard.provider.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/cupertino.dart';
+import 'package:project/config/inject.config.dart';
+import 'package:project/core/app_provider.dart';
+import 'package:project/core/exceptions/base.exception.dart';
+import 'package:project/modules/job/models/job.model.dart';
+import 'package:project/modules/job/services/job.service.dart';
+
+class JobsDashboardProvider extends AppProvider {
```

Comment:
Aici as incerca sa gasesc alt nume pentru provider
Poate doar JobsProvider?

Practic un JobsProvider de genu sta la root-ul tuturor provider-urilor care au de-a face cu job-uri pentru ca reprezinta store-ul tututor job-urilor incarcate de user. In mod normal pana si JobsDetailsProvider ar trebui sa fie definit ca si un ProxyProvider al lui JobsProvider.

Pe scurt provider-u asta nu o sa tina doar date care tin de dashboard.